VARIABLES.md

```diff
@@ -702,15 +734,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### kafka_connect_user
 
-Only use to customize Linux User Connect Service runs with. User must exist on host.
+Only use to customize Linux User Connect Service runs with. Default user is cp-kafka-connect
```

Comment:
Modify this variable to customize the Linux User that the Connect Service runs with. Default user is cp-kafka-connect.